camelComponent/src/main/java/io/vantiq/extsrc/camel/VantiqConsumer.java

```diff
@@ -46,6 +47,7 @@ public VantiqConsumer(VantiqEndpoint endpoint, Processor processor) {
     protected void doStart() throws Exception {
         super.doStart();
         endpoint.startup();
+        hdrDupMap = endpoint.getHeaderDuplicationMap();
```

Comment:
I see that this is set, but where is it used?

Reply:
it's not -- incomplete refactoring.  Removed